support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-id.html

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
I have no strong opinion, but what is the difference? I mean, there is a `div` already added by core, so adding `p` inside `div` is fine. The final HTML output is this:

```html
<div class="help" style="display: block;">
  <p>
    Every <code>input</code> step has an unique ID. It is used in the generated URL to proceed or abort.
  </p>
  <p>
    A specific ID could be used, for example, to mechanically respond the input from some external process/tool.
  </p>
<div>
```